scripts/build.sh

```diff
@@ -71,4 +75,13 @@ if [ "${BOUNDARY_INSTALL_BINARY}x" != "x" ]; then
 fi
 
 # Done!
-echo "==> Done!"
+echo "==> Results:"
+ls -hl bin/
+
+# Print the version output for just linux_amd64.
+# Since we run this script in CI, and our CI build jobs run on linux runners, 
+# We can only check the version output for a subset of builds
+if [ "${GOOS}" == "linux" ] && [ "${GOARCH}" == "amd64" ]; then
```

Comment:
@tmessi I added your suggestion to check the version output here. Unfortunately, this has some limitations since ubuntu runners can only run linux_amd64 artifacts, and seg faults on all others.. but it does provide a little bit more visibility

Reply:
Makes sense